CHANGELOG.md

```diff
@@ -3,6 +3,73 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.12.0
+
+_Aug 25, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Support horizontal bar chart
```

Comment:
Oh nice #9992. Would have this been worth a bit more detail on the highlight section of the changelog? https://mui.com/x/react-charts/bars/#layout, or this a too small feature?

Reply:
Imho, if the library was already widely used and stable, this would have been a major addition.
In the current situation, we could have added a screen to give the impression that we are doing a lot of stuff, but it was not a big feature to build in itself.